### DISCUSSION TITLE
New HW revision of Reolink RLC-810A + new firmware

### DISCUSSION THREAD

**Discussion by Kveri** [AT0myks/reolink-fw-archive#21] [General]
Hello,

I bought one RLC-810A with hardware no. IPC_523128M8MP few weeks ago. Few days ago I bought a set of Reolink RLC-810A cameras again, and noticed they have have different hardware no. - IPC_56064M8MP. Also they have quite new firmware - v3.1.0.1782_23021503 instead of v3.1.0.1162_22072805. I'm not sure if these two different HW revisions are compatible or not. The cameras with new hardware no. are not listed on Reolink download page, but I contacted Reolink support to see if they can fix it, provide some explanation or ideally provide the new firmware for IPC_523128M8MP.

**Comment by AT0myks** [comment]
As far as I know different hardware versions are not compatible. It's not the first time a new one shows up without being mentioned anywhere, and I'm not sure they even give you the choice when you buy. This and the fact that they sometimes (always?) don't provide the firmware that's shipped with the hardware is very frustrating. I hope they fix this one day and give you a satisfying answer.

**Comment by Kveri** [comment]
I got a response from Reolink support. This is indeed a new HW revision of the same model RLC-810A. The firmwares are incompatible between the old and new HW revisions and they only have one firmware v3.1.0.1782_23021503 for the new HW revision - that's the reason why it isn't listed on their download page, because for now the only available firmware is the one which is shipped with the cameras.

**Reply by AT0myks** [reply]
I don't get why they don't make available the base firmware for their devices. It's sitting somewhere on some employee's computer, they just have to take a few minutes to upload it. It seems to me they are intentionally trying to prevent their customers from easily going back to a previous firmware version. At the very least their support appears to be allowed to send a previous version if you ask them. But in my opinion you shouldn't have to ask them. And it's not realistic to assume that everyone makes a backup of every firmware. I wish I knew what their reasoning is. Thanks for the follow-up.
